.circleci/config.yml

```diff
@@ -37,7 +34,12 @@ aliases:
         if [ -n "$CIRCLE_PR_NUMBER" ]; then
           .ci/build.sh
         fi
-            
+
+  - &stack_build
+    run:
+      name: Build dependencies and Clash itself with Stack
+      command: .ci/stack_build.sh
```

Comment:
Should this also have the `if [ -n "$CIRCLE_PR_NUMBER" ]; then`?

Reply:
Well, I wasn't planning on adding it to GitLab CI (because `diepenheim` is busy enough as it is), so I think we should always test it. But maybe we should add it to GitLab CI anyway, as having it only on Circle means our own branches don't get tested until they're a PR..